### ISSUE TITLE
Failed to find configured root

### ISSUE THREAD

**Issue by gnanakeethan** [levibostian/Shutter-Android#8]
I am trying to use a different line than the default 
`<files-path name="internal_pictures_files" path="Pictures/" />`

to something like 

`<files-path name="internal_pictures_files" path="images/" />`

and it causes app crash and subsequent relaunch.  How should I really use to save to different directory. 

FYI: I am a novice.

**Comment by levibostian** [comment]
Hello there, @gnanakeethan. Thanks for giving Shutter a try! 

It is no surprise that your app is crashing when you change: `<files-path name="internal_pictures_files" path="Pictures/" />` to `<files-path name="internal_pictures_files" path="images/" />`. 

This is because Shutter is hard coded into saving pictures in the `Pictures/` directory. Because you are using `images/`, Shutter throws an error as it can no longer save into the `Pictures/` directory. 

If you don't mind me asking, why would you like to use the directory `images/` instead of the default `Pictures/`?

**Comment by gnanakeethan** [comment]
Quite right. I actually didn't know that earlier. I actually wanted to save pictures took with different actions (button clicks) to different locations. 

Anyhow thanks for clarification. Is it possible to implement? I shall try to contribute if possible.

**Comment by levibostian** [comment]
`Quite right. I actually didn't know that earlier. I actually wanted to save pictures took with different actions (button clicks) to different locations.`

I never thought about that. Tell me more about why you want to save pictures in different locations. Or, would you be ok with having all of the images saved to 1 location, but they are named a different file name? 

I am asking you questions to get a better idea of what you are looking for. Then we can talk about how to implement this into Shutter. 

The more information you can tell me about your use case the better help it is to me.

**Comment by gnanakeethan** [comment]
@levibostian I am currently sorting out that issue. Actually it would have been easier if I store images in multiple places. The use case is that, I am building an app in which there are proof of pickup/delivery photos needed in case there is no one to digitally sign using (SignaturePad). 

I thought to place the photos sorted by the ID in a specific directory so that I can list them all and check with remote and push if necessary. 
Right now using a "_" separator and adding a suffix to the name.  👍

**Comment by levibostian** [comment]
Ah, ok. 

I can see how having the ability to store a photo in a specific directory would be helpful. 

I created a [new issue](https://github.com/levibostian/Shutter-Android/issues/9) about adding this feature to the library. If you want, @gnanakeethan, feel free to take that issue and make a pull request to add the ability 😄. 

I will close this issue as it has been solved. We can talk about adding the directory feature in [this new issue](https://github.com/levibostian/Shutter-Android/issues/9) from here on out. 

Thanks, @gnanakeethan!
